tutorials/provisional/mosaic.py

```diff
@@ -0,0 +1,295 @@
+"""
+"=======================================
```

Comment:
```suggestion
=======================================
```

Reply:
Ah, this explains the local failures I was getting right before I gave up and went to bed last night (and pushed on the hope that it was a local issue :( )